src/dev.c

```diff
@@ -211,18 +210,14 @@ fido_dev_open_rx(fido_dev_t *dev, int ms)
 			r = FIDO_ERR_INTERNAL;
 			goto fail;
 		}
-		int res;
-		if ( (res = fido_dev_get_cbor_info_wait(dev, info, ms)) != FIDO_OK) {			
+		if ((r = fido_dev_get_cbor_info_wait(dev, info,
```

Comment:
@martelletto I used res on purpose - so if the fallback is disabled, the function will return success.
if you switch to using r, and the fallback is enabled, you need to set r to FIDO_OK

Reply:
which happens in line 232, no? although i wouldn't object to explicitly setting it to `FIDO_OK` for clarity

Reply:
sorry I missed that :)